js/modules/k6/ws/ws_test.go

```diff
@@ -633,6 +633,17 @@ func TestErrors(t *testing.T) {
 		require.NoError(t, err)
 		assertSessionMetricsEmitted(t, metrics.GetBufferedSamples(test.samples), "", sr("WSBIN_URL/ws-close"), statusProtocolSwitch, "")
 	})
+
+	t.Run("wrong status code", func(t *testing.T) {
+		test.VU.StateField.Options.Throw = null.BoolFrom(false)
```

Comment:
```suggestion
       t.Parallel()
		test.VU.StateField.Options.Throw = null.BoolFrom(false)
```

Reply:
Only this will lead to races due ot how the test is written.

I have opened a [PR](https://github.com/grafana/k6/pull/2732) fixing the tests and other lint issues. 

This has some changes of being merged for v0.41.0 (as it has practically no functional changes). But unfortunately this likely won't make it as we are now in the freeze period for code changes for v0.41.0. 

I would still recommend that once we merge the lint fixes, you rebase on top of master and push here. 

You will be able to copy paste one of the other use cases to call `t.Parallel()` and the rest of the test setup and copy your logic there. 

Sorry for the inconvience :(